src/main/webapp/app/overview/course-overview.component.html

```diff
@@ -103,6 +110,64 @@
                 </div>
             </mat-sidenav>
             <mat-sidenav-content [ngClass]="{ 'sidenav-height-dev': !isProduction || isTestServer }">
+                <!-- Dropdown menu -->
+                <div class="dropdown">
```

Comment:
I do not think that this comment is needed
```suggestion
                <div class="dropdown">
```